packages/deployments/contracts/src/cli/init/config/mainnet/production.ts

```diff
@@ -290,6 +290,21 @@ export const MAINNET_PRODUCTION_INIT_CONFIG: InitConfig = {
         },
       },
     },
+    {
+      name: "BTRST",
+      canonical: {
+        domain: "6648936",
+        address: "0x799ebfABE77a6E34311eeEe9825190B9ECe32824",
+        decimals: 18
+      },
```

Comment:
You need to add cap here as well.

Reference Link: https://github.com/connext/monorepo/blob/79dae16d9f88f19cfd8a06d3a567899fc920ed92/packages/deployments/contracts/src/cli/init/config/mainnet/production.ts#L24